src/app/components/error/error.layout.tsx

```diff
@@ -0,0 +1,65 @@
+import { ReactNode } from 'react';
+import { Box, Flex, Text, color, Button } from '@stacks/ui';
+
+import { Title } from '@app/components/typography';
+import GenericError from '@assets/images/generic-error.png';
+
+interface ErrorProps {
+  body: string;
+  helpText: ReactNode[];
```

Comment:
I was thinking you'd past in the list items as an array. This works too though, but as you're passing in the whole thing, it should just be

```suggestion
  helpText: ReactNode;
```

Reply:
Yep, I did mean to map thru them as an array. Thx for catching.